enhancements/installer/single-node-installation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Single node installation
+authors:
+  - "@eranco74"
+reviewers:
+  - "@romfreiman"
+  - "@hardys"
+  - "@crawford"
+  - "@markmc"
+approvers:
+  - "@hardys"
+  - "@crawford"
+  - "@markmc"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "https://github.com/openshift/enhancements/pull/302"
+  - "https://github.com/openshift/installer/pull/3978"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+
+I have tried https://github.com/openshift/enhancements/pull/302
+
+
+# Single node installation
+
+Lunch “all-in-one” control plane using static pods packed in Ignition file.
+This enhancement can be the first step of what is described in https://github.com/openshift/enhancements/pull/302
+The plan is to create bootstrap like control plane using static pods for:
+etcd, kube-apiserver, kube-controller-manager & kube-scheduler.
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+Enable single node kubernetes cluster installation using static pods via Ignition.
+Add a new command to the openshift installer `create aio-ignition` that will get as input the install-config and generate aio.ign
+The aio.ign content will be similar to what we have in the bootstrap Ignition file with some changes around bootkube and kubelet services:
+- Bootkube will finish when it sets up the control plane without applying all the openshift manifests.
+- kubelet will be configured to be a part of the cluster.
+When booting an rhcos image with the aio.ign we will get a similar flow as we get in the bootstrap node and end up with basic control plane.
+   
+## Motivation
+
+Demonstrate a prototype of creating a simple static Ignition file that boots an RHCOS machine and launches a basic Kube control plane
+
+### Goals
+
+1. Create a single node cluster composed of static pods, similar to the installer bootstrap.
+
+### Non-Goals
+
+1. Single ignition config that can be used for multiple clusters
```

Comment:
I suspect that many will see this as valuable enough to build in from the start because of latency issues, but @crawford probably knows better.